server/desktop/src/main/java/dev/slimevr/desktop/Main.kt

```diff
@@ -212,3 +221,18 @@ fun provideFeederBridge(
 
 	return feederBridge
 }
+
+const val CONFIG_FILENAME = "vrconfig.yml"
+fun resolveConfig(): String {
+	// If config folder exists, then save config on relative path
+	if (Path("config/").exists()) {
```

Comment:
ngl having a folder for this feels a little weird, usually a file is used as far as I've seen, but this should work

Reply:
yeah but for windows you need to create a "type of file" and if you want to remove or change file extension you have to change a windows configuration and like its a pain in the ass qwq

Reply:
Understandable, that's alright then